src/configs/chalk.c42spec

```diff
@@ -1992,27 +1992,27 @@ end users.
     type:    string
     default: "/sys/class/dmi/id/board_vendor"
     hidden:  true
-    doc:     "Path where to check AWS board vendor if running in AWS EC2"
+    doc:     "Path where to check AWS board vendor if running in AWS, Google or Microsoft nodes"
   }
 
   field sys_product_path {
     # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/identify_ec2_instances.html
     type:    string
     default: "/sys/devices/virtual/dmi/id/product_uuid"
     hidden:  true
-    doc:     "Path where to check AWS product if running in AWS EC2"
+    doc:     "Path where to check product uuid for cloud nodes"
   }
 }
 
-singleton aws {
```

Comment:
why the rename?

as we officially add multiple cloud providers I think it makes sense to separate them in independant configs as there could be conflicting values

Reply:
Personally, I'd prefer to make it super easy to capture things as consistently as possible across multiple clouds, so this approach does work well for me. 

I think if we find something where cloud-specific configuration would be important to people, we can easily support that too, but realistically I can't think of anything anyway.